code/ControllerExtension.php

```diff
@@ -36,6 +36,39 @@ public function processReview($data, \Form $form){
         $this->owner->redirectBack();
 
     }
+    
+    public function averageRating() {
+    
+        if($rating = \Reviewable\Review::get()->filter('ObjectID', $this->owner->ID)){
+
+            $countRating = $rating->Count();
+            $ratingTotal = 0;
+
+            foreach($rating as $review) {
+
+                $ratingTotal += $review->Rating;
+            
+            }
+
+            if(round($ratingTotal / $countRating) == 0) {
+                return '<img src="/reviewable/images/star-off.png" alt="1" title="bad"> <img src="/reviewable/images/star-off.png" alt="2" title="poor"> <img src="/reviewable/images/star-off.png" alt="3" title="regular"> <img src="/reviewable/images/star-off.png" alt="4" title="good"> <img src="/reviewable/images/star-off.png" alt="5" title="gorgeous">';
```

Comment:
Is there a way to do this without returning html directly? It's muddying the controller and view layers :)